finger-exercises-react/react/tic-tac-toe/src/app/components/Routs/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+
+import App from '../App';
```

Comment:
This should be imported from `/screens`.

You can do the refactor that we talked about.
Instead of having this `App` component use `Game` and render `PrevGames` in it.